CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+Thank you for your interest in contributing to Glosario!
+We welcome contributions of all kinds.
+If you're here to submit a definition,
+either for a new term
+or in another language for an existing term,
+please take a moment to read the guidance below.
+
+## Contribution Workflow
+
+When you're making your contribution(s) to Glosario, please keep the following
+guidance in mind:
+
+1. **Check the [Issues page of the Glosario repository][issues]** to see what terms and definitions other contributors are already working on. This will help you avoid accidentally spending time writing the same term(s)/definition(s) as someone else.
+2. When you know what term or definition you'd like to add, tell others about it by writing **a [new issue] on the main Glosario repository.**
+3. **Add one new term or definition per Pull Request.** We'd love to receive multiple contributions from you but please keep each separate new term or definition in a distinct branch (see below) and Pull Request. This makes it much easier for the Glosario maintainers/editors to review your contributions. When you open the Pull Request you can refer to the Issue you opened before (see step 2),  e.g. if your Issue was number 49 you should write "Closes \#49." GitHub will then automatically close the Issue if and when your Pull Request is merged.
+4. When making multiple contributions, please **[create a new branch][github-branches] for each term/definition.** Add the new term/definition on a separate branch, created from the `master` branch, and make a new Pull Request each time. This will help you avoid accidentally combining multiple contributions in a single Pull Request.
+
+## 1. Adding a new term to the glossary
+
+To add a new entry, please [fork][forking-guide] the [main Glosario repository][repo] and, on a new branch, add the term and definition to [`glossary.yml`][glossary]. This glossary file is written in [YAML].
+
+Here is an example of how your glossary entry should be structured:
+
+```
+- slug: cran
+  ref:
+    - base_r
+    - tidyverse
+  en:
+    term: "Comprehensive R Archive Network"
+    acronym: "CRAN"
+    def: >
+      A public repository of R [packages](#package).
+```
+
+-   The value associated with the `slug` key identifies the entry.
+    -   It must be unique within the glossary.
+    -   It must be in lower case and use only letters, digits, and the underscore
+        (to be compatible with Jekyll's automatic slug creation).
+-   The entry *may* have a `ref` key.
+    If it is present,
+    its value must be a list of identifiers of related terms in this glossary.
+-   Every other top-level key must be a two-letter ISO 639 language code such as `en` or `fr`.
+  (Refer to the "639-1" column of [this table][iso639-table-en].)
+    -   Every entry must have at least one such language section.
+-   Within each language section for each term:
```

Comment:
Note that the keys like `term` and `acronym` are _not_ to be translated: they must be as-is. (We've had at least one person use the Spanish word for 'term'.)